amazonei_setup.py

```diff
@@ -568,12 +568,11 @@ def is_ei_service_available(self,region_name):
 
 
     def get_accelerator_type(self):
+        accel_types = ['eia2.medium','eia2.large','eia2.xlarge','eia1.medium','eia1.large','eia1.xlarge']
+        accel_memory = ['2 GB of accelerator memory','4 GB of accelerator memory','8 GB of accelerator memory','1 GB of accelerator memory','2 GB of accelerator memory','4 GB of accelerator memory']
 
-        accel_types = ['eia1.medium','eia1.large','eia1.xlarge']
-        accel_memory = ['1 GB of accelerator memory','2 GB of accelerator memory','4 GB of accelerator memory']
-
-        print("\nPlease type index of the accelerator type to use:\n")
-
+        print("\nPlease type index of the accelerator type to use: (Note: eia1s are in deprecation mode. Please select eia2s (eia2s cost less than eia1s and have double the memory) unless there is a good reason not to. However, eia2s are currently NOT available in ap-northeast-1.)\n")
```

Comment:
This message has nested parentheses. Probably better to rephrase a bit.

Reply:
Fixed it.